tests/keyboard_injected.rs

```diff
@@ -14,6 +14,13 @@ mod keyboard_hook_tests {
     use willhook::event::IsSystemKeyPress::*;
     use mki::Keyboard;
 
+    #[test]
```

Comment:
Please expand the test before merging.

Test like this would suffice:
```
imitate key press
imitate other key press
imitate mouse button
imitate mouse move
recv() accordingly to drain all events
call try_recv() to validate the events queue is empty
```

Dedicated file "blocking.rs" would be good.